automation/jleechanorg_pr_automation/tests/test_pr_monitor_feedback_loop.py

```
@@ -114,13 +116,146 @@ def test_all_automation_markers_recognized(self, monitor):
+                "createdAt": "2026-02-09T08:00:00Z",
+            },
+            # Bot comment after automation comment (would trigger "new bot comment" detection)
+            {
+                "body": "Please fix the SQL injection vulnerability",
+                "author": {"login": "coderabbitai[bot]"},
+                "createdAt": "2026-02-09T08:05:00Z",
+            },
+        ]
+
+        # Check if comment exists for this SHA
+        has_comment = monitor._has_codex_comment_for_commit(comments, head_sha)
```

Ruff (0.14.14)

[error] Private member accessed: `_has_codex_comment_for_commit`

(SLF001)